### ISSUE TITLE
Hangouts widget doesn't work

### ISSUE THREAD

**Issue by d4rky-pl** [google/instant-hangouts#13]
Example HTML file:

``` html
<script src='instanthangouts-0.1.0.js' async></script>
<div
    class='instanthangouts'
    lang='fr'
    parsetags='onload'
    publisher_id='112744459749475398119'
    render='hangout'
    room_id='my_room'
    topic='My Topic'
    width='200'></div>
```

Result: trying to create iframe with url `https://talkgadget.google.com/u/0/talkgadget/_/widget?hangout_type=normal&publisher_id=112744459749475398119&render=hangout&room_id=my_room&topic=My%20Topic&widget_size=136&width=200&hl=fr&origin=http%3A%2F%2Fplayground.nerdblog.pl&gsrc=3p&jsh=m%3B%2F_%2Fscs%2Fapps-static%2F_%2Fjs%2Fk%3Doz.gapi.en.Jf3CPDDPOEs.O%2Fm%3D__features__%2Fam%3DAQ%2Frt%3Dj%2Fd%3D1%2Ft%3Dzcms%2Frs%3DAItRSTONGn74v8Rs_7Fvo8yzjgTE3aOpqg` resulting in **400 Bad Request**

URL to example: http://playground.nerdblog.pl/hangouts.html

**Comment by chrisgraham** [comment]
Indeed. It simply does not work. The only thing that works is the standard button, which doesn't need this library to create.

This reflects very poorly on Google. They need to at minimum announce this library no longer works.
